src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/collectionsgrid/view/CollectionsGrid.kt

```diff
@@ -44,9 +44,12 @@ class CollectionsGrid : Fragment() {
                 card {
                     addClass(DefaultStyles.defaultCard)
                     cardfront {
+                        isCompleteProperty.bind(item.second.booleanBinding{it == 1.0})
```

Comment:
equality with floating point numbers has the potential to be buggy;

if 1.0 is meant to mean 100% I would recommend just using integers 0-100

Reply:
the reason I did that is because the tornadofx progress bar takes in a double value from 0-1 to indicate progress

Reply:
you should do some sort of epsilon check then; basically it >= .9999